hack/kube-api-linter/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+options:
+  no_parent_owners: true
+reviewers:
+  - api-approvers
```

Comment:
FYI @kubernetes/api-approvers -- this will cover the configuration of which api linters are enabled and which exceptions are permitted to those lints (see the two files below)

Currently none are enabled in either case.